lib/deprecation_toolkit/warning.rb

```diff
@@ -2,6 +2,8 @@
 
 module DeprecationToolkit
   module Warning
+    extend self
```

Comment:
I think this will now work because there is no `super` to call in the `else` on < 2.5